lib/src/rules/prefer_if_null_operators.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/dart/element/element.dart';
```

Comment:
I think this is unused?

Removing it should fix the travis break:

https://travis-ci.org/dart-lang/linter/jobs/532359228

Reply:
Oops, thanks